calls.go

```diff
@@ -138,15 +141,24 @@ func (call *callsCmd) list(ctx *cli.Context) error {
 		params.PerPage = &per_page
 	}
 
-	resp, err := call.client.Call.GetAppsAppCalls(&params)
-	if err != nil {
-		switch e := err.(type) {
-		case *apicall.GetAppsAppCallsNotFound:
-			return errors.New(e.Payload.Error.Message)
-		default:
-			return err
+	for {
+		resp, err := call.client.Call.GetAppsAppCalls(&params)
+		if err != nil {
+			switch e := err.(type) {
+			case *apicall.GetAppsAppCallsNotFound:
+				return errors.New(e.Payload.Error.Message)
+			default:
+				return err
+			}
 		}
+
+		printCalls(resp.Payload.Calls)
+
+		if resp.Payload.NextCursor == "" {
+			break
```

Comment:
don't think this is really such a good idea. when we have real users, they are going to have thousands, tens of thousands, maybe millions of calls and this will make __a ton__ of api calls, take a _really_ long time for the CLI command to return and is very likely not what the user wants. we should allow a user to be able to get this data (and we do, through the API), but we absolutely should not make it the default behavior [or even make it all that easy]. it seems reasonable as a default to return the last hour of calls, but not every single call.

Reply:
even last hour is a lot, really. but i think from the CLI using time ranges makes sense. most of the time, at least, when i'm searching calls i want ones from last few hours (from some test), not some 'last N' calls (which is harder to gauge).

Reply:
Okay, PR updated, i tend to agree that last hour calls may work but with pretty intensive Fn usage it may be hit the same problem, but i do believe that users will use pagination opts for that.

Reply:
thanks. we can adjust, some bound is better than none. for me, usually looking at last few calls is all i want, and if i want to do something with a lot of data i end up using the api client in go to write a little program, but that's me. 

i'm skeptical of apps / routes listing all by default too. it defeats having pagination to begin with (same with calls). i'd be slightly more comfortable with truncating it to 100 or 1000 and forcing a user to paginate, b/c users can do dumb things sometimes (make 1M routes...). one of those things where i don't think it would be the end of the world to truncate and output that the user should use the client bindings / api to sift through all the data and not the cli, cli pagination by user just seems gnarly and to be discouraged.

Reply:
I think any listing call should just return a fixed number (ie: per-page). Agree with Reed that it definitely should not auto paginate through all results, there's almost no scenario where that would be a good idea. We could use the 'per-page' flag though to page through multiple result sets though. eg: if user passed in `--per-page 1000`, then loop until 1000 results.